src/components/Requirements.vue

```diff
@@ -1,6 +1,13 @@
 <template v-if="semesters">
   <div class="requirements">
-    <div class="fixed">
+    <div id="req-tooltip" class="fixed"
```

Comment:
Let's not keep this div if it is not going to show up at all on CoursePlan right now? Add a comment here and comment out if you think the code is worth preserving, otherwise delete it.